src/components/TagCloud.js

```diff
@@ -134,7 +137,8 @@ export default class TagCloud extends React.Component<TagCloudDefaultProps, TagC
       };
       const className = TagCloud.getClassNameForLevel(size);
       return (
-        this.props.noLink ? (<li key={tcv.label}>
+        (this.props.noLink || tcv.noLink) ?
+        (<li key={tcv.label} className="attivio-cloud-noLink">
```

Comment:
Please reformat this so the opening parenthesis wrapping the JSX isn't on the same line as the JSX (e.g. the opening `<li>` tag). The updated ESLint will complain about this and it'll be easier to fix it now... The same is true for the "else" part. (You'll be able to put the condition on the same line as the `return` statement, to make it more compact.... and the extra set of parentheses that are around the whole return value for someone reason. :-) )
```
return (this.props.noLink || tcv.noLink) ? (
  <li key={tcv.label} className="attivio-cloud-noLink">
    <span className={className}>
      {tcv.label}
    </span>
  </li>
) : (
  <li key={tcv.label}>
    <a className={className} onClick={callback} role="button" tabIndex={0}>
      {tcv.label}
    </a>
  </li>
);
```